prepare-kubernetes/mariadb.md

```diff
@@ -7,34 +7,131 @@ layout: page
 ## Install MariaDB Helm Chart
 
 Download and edit values.yaml for frappe related mariadb config.
+  Source : [Mariadb Config Yaml](https://raw.githubusercontent.com/bitnami/charts/master/bitnami/mariadb/values.yaml)
+<details>
```

Comment:
This values will become outdated soon.

Most people just copy it from here.

I think there should be a disclaimer here. No problem with repeating warning messages and disclaimers only few people really read them.

I was thinking if it's okay to use a marquee tag just to make the disclaimer visible 🤦

Reply:
Will remove the mariadb yaml file, the source has been mentioned anyway and ad a warning to to the latest/platform relevant instructions from the source.
What are the changes that need to be made for the NFS, post k8 resources. ?

Reply:
https://helm.erpnext.com/kubernetes-resources/create-site-ingress ingress needs to change from `extensions/v1beta1` to `networking.k8s.io/v1`

Rest everything is good.

Reply:
I completely forgot this new addition that can be helpful. Check values for `createSite: {}` and `ingress: {}`, If everything is right it will create a site and ingress after installation of the helm chart. If `createSite.dropSiteOnUninstall` is true it'll even clean up the site on uninstall of helm chart. You can make it part of docs. The only problem with this is, if the hooks fail, debugging is difficult for newcomers.